modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -935,6 +936,28 @@ const register = (editor: Editor): void => {
     }
   });
 
+  const documentsFileTipesOptionsSchema = StructureSchema.arrOfObj([
+    FieldSchema.requiredString('mimeType'),
+    FieldSchema.requiredArrayOf('extensions', StructureSchema.valueOf((ext) => {
+      if (Type.isString(ext)) {
+        return Result.value(ext);
+      } else {
+        return Result.error('Extensions must be an array of strings');
+      }
+    })),
+  ]);
+
+  registerOption('documents_file_types', {
+    processor: (value: unknown) => StructureSchema.asRaw<DocumentsFileTypes[]>('documents_file_types', documentsFileTipesOptionsSchema, value).fold<DocumentsFileTypesProcessorReturnType>(
+      (_err) => ({
+        valid: false,
+        message: 'Must be a non-empty array of objects matching the configuration schema: https://www.tiny.cloud/docs/tinymce/latest/content-filtering/#documents_file_types' // TODO: this is a placeholder
+      }),
+      (val) => ({ valid: true, value: val })
+    ),
+    default: []
```

Comment:
This is an invalid default, since the processor requires a non-empty array. The AI picked this up where you used it in the plugin.